__tests__/aticles.test.js

```diff
@@ -186,4 +186,25 @@ describe("Articles End Point", () => {
         });
     });
   });
+  test("Get :200 all articles objects in the database that has topic", () => {
+    return request(app)
+      .get("/api/articles")
+      .query({ topic: "mitch" })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(4);
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("Get:400  when topic does not exist", () => {
+    return request(app)
+      .get("/api/articles")
+      .query({ topic: "sanae" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Articles not found");
+      });
+  });
```

Comment:
One untested behaviour here:

- Status 200, valid topic query, but has no articles responds with an empty array of articles, e.g. ?topic=paper.